app/src/main/java/com/cdio/solitaire/model/Card.kt

```diff
@@ -0,0 +1,46 @@
+package com.cdio.solitaire.model
+
+enum class Suit {
+    NA, CLUBS, DIAMONDS, HEARTS, SPADES; // Note the order: C = 1, D = 2, H = 3, S = 4.
+
+    fun short(): String {
+        return when (this.ordinal) {
+            1 -> "C"
+            2 -> "D"
+            3 -> "H"
+            4 -> "S"
+            else -> "NA"
+        }
+    }
+
+    fun isRed(): Boolean {
+        return when (this) {
+            DIAMONDS, HEARTS -> true
+            else -> false
```

Comment:
I think making the return value as explicit as possible makes the code more readable, for example:
```suggestion
    enum class Color {
        RED,
        BLACK
    }

    fun color(): Color {
        return when (this) {
            DIAMONDS, HEARTS -> Color.RED
            else -> Color.BLACK
```

However it also makes the class a bit more bloated, so it's up to you. 🙂